opensoundscape/raven.py

```diff
@@ -159,51 +163,56 @@ def query_annotations(directory, cls, col="class", print_out=False):
     return output
 
 
-def split_single_annotation(
-    raven_path, split_len_s, total_len_s=None, species=None, col="class"
-):
-    """Split a Raven selection table into short annotations
+def split_starts_ends(raven_file, col, starts, ends, species=None, min_label_len=0):
+    """Use a list of start and end times to split a Raven files
```

Comment:
does this function aggregate annotations (from one column of a Raven annotation file) over each start-end time segment into one-hot labels? It's hard to tell from the docstring

Reply:
Good suggestion; added more descriptive docstring